content/learn/book/welcome/app/_index.md

```diff
@@ -0,0 +1,90 @@
++++
+title = "Creating Bevy apps"
+weight = 2
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+When you're making a game in Bevy, you will have a single `App`, which stores and handles all of your game logic and data.
+Let's take a closer look at that hello world code.
+
+```rust
+use bevy::prelude::*;
+
+fn main(){
+  App::build()
+  .add_system(hello.system())
+  .run();
+}
+
+fn hello(){
+    println!("Hello, Bevy!")
+}
+```
+
+So, we first create our [`App`](https://docs.rs/bevy/latest/bevy/app/struct.App.html) by telling it to [`build`](https://docs.rs/bevy/latest/bevy/app/struct.App.html#method.build), performing initialization steps.
+This produces an [`AppBuilder`](https://docs.rs/bevy/latest/bevy/app/struct.AppBuilder.html), which we can use to configure the `App`.
+We do so by add a simple system, which prints "Hello, Bevy!" when it is run.
+Finally once all of our work is complete, we call [`AppBuilder::run()`](https://docs.rs/bevy/latest/bevy/app/struct.AppBuilder.html#method.run) to actually make our app *do things*.
+
+## What makes an App?
+
+So, what sort of data does are `App` really store?
+Turns out: it's fairly simple.
+Looking at the docs for [`App`](https://docs.rs/bevy/latest/bevy/app/struct.App.html), we find three fields:
+
+```rust
+struct App {
+  world: World
+  runner: Box<dyn Fn(App) + 'static, Global>
+  schedule: Schedule
+}
+```
+
+The `World` stores all of our data, both entity-component data and resources.
+The `runner` field contains a function that tells our app how to loop and execute.
+And the `Schedule` contains all of the systems that execute on our data, stored in a way that our scheduler can use to run them at the appropriate time.
```

Comment:
This is better suited as API docs. Especially as it is stated later:

> Generally, you'll be operating at a higher-level of abstraction than this

We can transfer this knowledge to the API docs, which are currently scarce. There is also a benefit in doing this: pointing users to the API docs will encourage them to to it by their own.